changelog/unreleased/enhancement-introduce-web-app-user-management

```diff
@@ -0,0 +1,6 @@
+Enhancement: Introduce web-app-user-management app
```

Comment:
```suggestion
Enhancement: Introduce user-management app
```
User-facing the app name is "user-management" / "User management". `web-app-user-management` is only the package name.